launcher/core/injector/preloadinjector.cpp

```diff
@@ -51,13 +55,15 @@ bool PreloadInjector::launch(const QStringList &programAndArgs, const QString &p
 {
     Q_UNUSED(probeFunc);
 
+    const QString actualProbeDll = m_probeDllOverride.isEmpty() ? probeDll : m_probeDllOverride;
```

Comment:
This would be better preserved at the caller side of this function, i.e. in Launcher::start. That requires the use of a separated commandline option though.